src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -124,7 +124,7 @@ function Create-VirtualMachine($rgname, $vmname, $loc)
 
     # NRP
     $subnet = New-AzureRmVirtualNetworkSubnetConfig -Name ('subnet' + $rgname) -AddressPrefix "10.0.0.0/24";
-    $vnet = New-AzureRmVirtualNetwork -Force -Name ('vnet' + $rgname) -ResourceGroupName $rgname -Location $loc -AddressPrefix "10.0.0.0/16" -DnsServer "10.1.1.1" -Subnet $subnet;
+    $vnet = New-AzureRmVirtualNetwork -Force -Name ('vnet' + $rgname) -ResourceGroupName $rgname -Location $loc -AddressPrefix "10.0.0.0/16" -Subnet $subnet;
```

Comment:
I decided to remove the DNS server parameter after comparing these test cases against their equivalents in the dev branch. This parameter value was removed from the test cases in the dev branch as part of [commit cfa9f94e06f7925af4bfc1511bfa05779d90d967](https://github.com/Azure/azure-powershell/commit/cfa9f94e06f7925af4bfc1511bfa05779d90d967), so the dev branch is not specifying this parameter in these test cases. Is it really that important to exercise that code path in the stack-dev branch, when it isn't exercised by these test cases in the dev branch?

Also, I know that the TestSingleNetworkInterfaceDnsSettings test case, at least, will still specify a DNS server parameter after these changes, so this code path won't go completely untested.